app/core/tests/test_sandbox_runner.py

```
@@ -0,0 +1,70 @@
+                "session_dir": str(session_dir),
+                "allowed_input_paths": [str(input_file)],
+                "timeout_seconds": 5,
+                "cpu_seconds": 5,
+                "memory_bytes": 128 * 1024 * 1024,
+                "file_size_bytes": 10 * 1024 * 1024,
+            }
+        ),
+        encoding="utf-8",
+    )
+
+    completed = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)